collector/aws/resources/rds.go

```diff
@@ -136,9 +153,50 @@ func (r *RDSManager) Detect(metrics []config.MetricConfig) (interface{}, error)
 					"formula_value":       formulaValue,
 					"name":                *instance.DBInstanceIdentifier,
 					"instance_type":       *instance.DBInstanceClass,
+					"engine":              *instance.Engine,
 					"region":              r.awsManager.GetRegion(),
 				}).Info("RDS instance detected as unutilized resource")
 
+				instancePricingFilters := r.getPricingInstanceFilterInput(instance)
+				instancePrice, err := r.awsManager.GetPricingClient().GetPrice(instancePricingFilters, "", r.awsManager.GetRegion())
+				if err != nil {
+					log.WithError(err).Error("Could not get rds instance price")
+					continue
+				}
+
+				var hourlyStoragePrice float64
+				if rdsStorageType, found := rdsStorageType[*instance.StorageType]; found {
+					var storagePricingFilters pricing.GetProductsInput
+					switch *instance.Engine {
+					case "aurora", "aurora-mysql", "aurora-postgresql":
+						storagePricingFilters = r.getPricingAuroraStorageFilterInput(rdsStorageType, pricingRegionPrefix)
+					default:
+						deploymentOption := r.getPricingDeploymentOption(instance)
+						storagePricingFilters = r.getPricingRDSStorageFilterInput(rdsStorageType, deploymentOption)
+					}
+
+					log.WithField("storage_filters", storagePricingFilters).Debug("pricing storage filters")
+					storagePrice, err := r.awsManager.GetPricingClient().GetPrice(storagePricingFilters, "", r.awsManager.GetRegion())
+					if err != nil {
+						log.WithError(err).Error("Could not get rds storage price")
```

Comment:
what do you say to add the filter object into the log to improve the troubleshoot